theano/sparse/tests/test_basic.py

```diff
@@ -484,7 +484,7 @@ def test_adv_sub1_sparse_grad(self):
 
         # Test the sparse grad
         valm = numpy.random.rand(5, 4).astype(config.floatX)
-        valv = numpy.random.random_integers(0, 4, 10)
+        valv = numpy.random.randint(0, 4, 10)
```

Comment:
Since `randint` *excludes* the higher bound, but `random_integers` included it, I think the proper translation should be `numpy.randint(0, 5, 10)`.
It may not be a big deal, but I would prefer if we kept the same values.